### PR TITLE
[Synthetics] Tunnel: Reduce `Proxy opened` log verbosity

### PR DESCRIPTION
**Pull request by lefebvree**
### What and why?

Only display `[Tunnel] Proxy opened for test ...` log once per test instead on every proxy connection to reduce output spam for browser tests.

### Review checklist

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration)

### CODE REVIEW

File: src/commands/synthetics/tunnel.ts [hunk 1]
```diff
@@ -142,7 +143,11 @@ export class Tunnel {
     }
 
     // Username is allowed and key authentication was successful
-    this.log(`Proxy opened for test ${user}`)
+    if (!this.openedTunnels.has(ctx.username)) {
+      // Limit to one log per test
+      this.openedTunnels.add(ctx.username)
+      this.log(`Proxy opened for test ${ctx.username}`)
+    }
```
Comment (etnbrd) [inline src/commands/synthetics/tunnel.ts:150]: Nit, maybe removing the `ctx.username` at the end of the life-cycle of the tunnel?
Another solution would be to throttle based on the username?

**Review by etnbrd — APPROVED** [review]
LGTM

File: src/commands/synthetics/tunnel.ts (hunk 1 above)
Comment (lefebvree) [reply src/commands/synthetics/tunnel.ts:150]: The life-cycle of the tunnel is basically the process itself, there won't be more than one opened in the command life-cycle so I think we're good to leave them there

Comment (m-rousse) [reply src/commands/synthetics/tunnel.ts:150]: These logs will not bring much value to an user at a test level (it is unlikely that one test amongst a suite will not have its tunnel opened), could we make it be displayed only once or at most once per tunnel?

**Review by m-rousse — APPROVED** [review]

**Pull request merged**